cfg_mgr.lua

```diff
@@ -44,20 +62,25 @@ local function set_video_format()
         self.config.snapshot_extension = '.jpg'
         self.config.snapshot_codec = 'mjpeg'
     end
-    -- Animated webp images can only have .webp extension.
-    -- The user has no choice on this.
```

Comment:
Lost the comment here.

Reply:
I thought this comment meant that there is only one animation format, compared to three formats for still pictures. Since I've added support for another animation format, I've removed the comment.

If the comment was actually supposed to state that WebP animations come with and only with `.webp` extensions, well, it's kinda obvious.